src/drawer.multicanvas.js

```diff
@@ -212,6 +234,24 @@ export default class MultiCanvas extends Drawer {
         this.canvases.push(entry);
     }
 
+    addCanvases() {
```

Comment:
can you add a docstring for this new method?

Reply:
Absolutely. I meant to go back and update this, and just forgot.